CSS/style.css

```diff
@@ -0,0 +1,240 @@
+div {
+  display: block;
+}
+
+body {
+  margin: 0;
+}
+
+nav {
+  display: block;
+}
+
+header {
+  background-image: url(../images/hero-image.jpg);
+  background-position: 5px -744px;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 570px;
+  margin-left: auto;
+  margin-right: auto;
+}
```

Comment:
The CSS background should cover the whole page irrespective of the screen size. you can see that in big screens the background and the whole page goes to the left you should make it cover the whole page by using background-size: cover;, here what I have:
![image](https://user-images.githubusercontent.com/34653764/85211863-32865080-b345-11ea-804a-5189d4bcc538.png)

Reply:
Tried adding the suggested property. I was creating structure mismatch! So with little work around tried fixing. Thanks!